src/qnetvo/cost/negativity.py

```diff
@@ -0,0 +1,69 @@
+import pennylane as qml
+from pennylane import math
+from pennylane import numpy as np
+from ..qnodes import reduced_density_matrix_qnode
+
+def negativity_cost_fn(network_ansatz, m, n, wires, qnode_kwargs={}):
+    """Constructs an ansatz-specific negativity cost function.
+
+    Negativity can be used to identify if two subsystems :math:`A` and :math:`B` are entangled, through the PPT criterion. Negativity is an upper bound for distillable entanglement.
+
+    This entanglement measure is expressed as
+
+    .. math::
+
+            \\mathcal{N}(\\rho) = |\\sum_{\\lambda_i < 0}\\lambda_i|
+
+    where :math:`\\rho^{\\Gamma_B}` is the partial transpose of the joint state with respect to the :math:`B` party, and :math:`\\lambda_i` are all of the eigenvalues of :math:`\\rho^{\\Gamma_B}`
+
+    :param ansatz: The ansatz circuit on which the negativity is evaluated.
+    :type ansatz: NetworkAnsatz
+
+    :param m: The size of the :math:`A` subsystem.
+    :type m: int
+
+    :param n: The size of the :math:`B` subsystem.
+    :type n: int
+
+    :param wires: The wires which define the joint state.
+    :type wires: list[int]
+
+    :param qnode_kwargs: Keyword arguments passed to the execute qnodes.
+    :type qnode_kwargs: dictionary
+
+    :returns: A cost function ``negativity_cost(*network_settings)`` parameterized by
+              the ansatz-specific scenario settings.
+    :rtype: Function
+    """
```

Comment:
Please add a `:raises ValueError:` in the docstring to document the constraints on `m` and `n`

As an example, you can see docstring for the `qnetvo.NetworkAnsatz` class.